rfcs/address_space_overrides.md

```diff
@@ -0,0 +1,111 @@
+# RFC #72: Address space overrides
+
+## Summary
+
+The goal of this RFC is to enable web platform tests to cover the
+[CORS-RFC1918](https://wicg.github.io/cors-rfc1918) specification entirely.
+More specifically, to enable tests to exercise user agent behavior in the face
+of responses served from `local`, `private` and `public`
+[address spaces](https://wicg.github.io/cors-rfc1918#address-space).
+
+This is achieved by passing command-line flags to the browser under test forcing
+it to artificially consider specific IP addresses `local`, `private` or
+`public`.
+
+This RFC aims to fix
+[web-platform-tests/wpt#26166](https://github.com/web-platform-tests/wpt/issues/26166).
+
+## Details
+
+### Address allocation
+
+For the purposes of testing, declare that the following subnets are not `local`
+but instead:
+
+* 127.1.0.0/16: `private`
+* 127.2.0.0/16: `public`
+
+### Browser changes
+
+A new command-line flag (or a collection thereof) is added to each browser (it
```

Comment:
It would be useful to file bugs on various browsers to ensure that this plan is actually plausible. Unlike most RFCs, the majority of the implementation complexity here is outside wpt itself.

Reply:
Also fair. I'm curious if you have any feedback on this approach vs the "Use individual IP addresses" alternative?

Reply:
Resolving this thread - discussion with browser vendors continues on the PR and in linked issues.